### DISCUSSION TITLE
headerHeight不生效

### DISCUSSION THREAD

**Discussion by monadaa** [umijs/umi#10240] [Q&A]
app.tsx
export const layout: RunTimeLayoutConfig = () => {
  return {
    title: '配置中心',
    headerHeight: 50,
    layout: 'mix', 
    splitMenus: false,
    logo: 'https://img.alicdn.com/tfs/TB1YHEpwUT1gK0jSZFhXXaAtVXa-28-27.svg',
    menu: {
      locale: false,
    },
    // 默认布局调整
    rightContentRender: () => <RightContent />,
    footerRender: () => <Footer />,
    menuHeaderRender: undefined,
  };
};

headerHeight设置后不生效了

**Comment by fz6m** [comment]
我在 https://procomponents.ant.design/components/layout 搜了一下，有 headerHeight 这个选项么
